packages/features/insights/server/insightsDateUtils.ts

```
@@ -0,0 +1,206 @@
+  end: dayjs.Dayjs;
+  timeView: TimeViewType;
+  wholeStart: dayjs.Dayjs;
+  wholeEnd: dayjs.Dayjs;
+}): string => {
+  const omitYear = wholeStart.year() === wholeEnd.year();
+
+  switch (timeView) {
+    case "day":
+      return omitYear ? start.format("MMM D") : start.format("MMM D, YYYY");
+    case "week":
+      const startFormat = "MMM D";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)